src/chronify/models.py

```diff
@@ -74,6 +74,27 @@ def list_columns(self) -> list[str]:
         return super().list_columns() + [self.value_column]
 
 
+class PivotedTableSchema(TableSchemaBase):
+    """Defines the schema for an input table with pivoted format."""
+
+    pivoted_dimension_name: str = Field(
+        default=None,
+        description="Use this name for the column representing the pivoted dimension during "
+        "an unpivot operation.",
+    )
+    value_columns: list[str] = Field(description="Column in the table that contain values.")
```

Comment:
Singular column right?

Reply:
Are you just pointing out that the description should have `Columns`? There will definitely be multiple value columns.

Reply:
I was thinking of the unpivoted format, but this is pivoted, so yeah just the description.